core/src/data/value/convert.rs

```diff
@@ -17,7 +17,7 @@ impl From<&Value> for String {
         match v {
             Value::Str(value) => value.to_owned(),
             Value::Bytea(value) => hex::encode(value),
-            Value::Bool(value) => (if *value { "TRUE" } else { "FALSE" }).to_owned(),
+            Value::Bool(value) => (if *value { "true" } else { "false" }).to_owned(),
```

Comment:
It looks that you are using `Value` -> `String` for serialize data, but it would be better to use json module.
ref. https://github.com/gluesql/gluesql/blob/main/core/src/data/value/json.rs